app/containers/LanguageProvider/tests/index.test.js

```diff
@@ -10,27 +10,26 @@ import { translationMessages } from '../../../i18n';
 
 const messages = defineMessages({
   someMessage: {
-    id: 'some.id',
+    id: 'boilerplate.containers.LanguageProvider.index.test',
     defaultMessage: 'This is some default message',
-    en: 'This is some en message',
```

Comment:
It's my understanding that `react-intl` doesn't work like this for adding translations. I actually feel this format makes more sense than having a translation file for each locale but maybe I'm in the minority.

Because of this, I changed the `id` to match the format found in `app/translations/en.json`